tooling/generate_instances_types.py

```diff
@@ -0,0 +1,18 @@
+import boto3
+
+session = boto3.Session(region_name="us-east-1", profile_name="sch-gov")
```

Comment:
Profile name should be passed

Reply:
Sorry @joaoqalves I can do it better, it's only a quick tool to help me to generate the instances type enum...

Reply:
@joaoqalves done!